build.sbt

```diff
@@ -11,9 +11,8 @@ lazy val root = (project in file("."))
   .aggregate(one, two)
   .dependsOn(one, two)
 
-scalaVersion := "2.12.4"
-
-crossScalaVersions := Seq("2.11.12", "2.12.4")
+crossScalaVersions := Seq("2.12.7", "2.11.12")
```

Comment:
Do we need to maintain the `crossScalaVersions`? Actually, why do we need it in the first place?

Reply:
that's a good point, why we are cross-building?

I looked at the history and I don't see any clear reason. I think it was just a reflex of adding it when we started to cross build to 2.11 and 2.12

Reply:
Ok, I think that if it works we could _not_ remove it and add an issue to clean this up later.

Reply:
Ah, just seen your new commit. LGTM too. :-)